lighty-tests-report/pom.xml

```diff
@@ -97,6 +97,10 @@
             <groupId>io.lighty.kit.examples.controllers</groupId>
             <artifactId>lighty-guice-app</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This commit is about adding lighty-community-restconf-actions-app to lighty-test-report. Not about
`Add lighty-community-restconf-actions-app to lighty-bom`

Reply:
placed to a separate commit